data/governments.txt

```diff
@@ -788,6 +898,7 @@ government "Quarg"
 		"Hai (Wormhole Access)" .01
 		"Pirate" -.01
 		"Pirate (Devil-Run Gang)" -.01
+		"Gegno" 0.1
```

Comment:
```suggestion
		"Gegno" 0.1
		"use foreign penalties for"
			"Gegno"
```
this is needed, otherwise you can refuel Gegno to please Quarg

Reply:
Idk about this - on stream Derpy said this was a "no". I could have interpreted this the wrong way though. @Amazinite thoughts?

Reply:
I'm confused does he want ppl to use Gegno to farm Quarg rep with ease by refueling civilian high crew ships?
Caus that's all this would change since default values are used for the rest

Reply:
I said no to the 1:1 rep. Didn't comment on the foreign penalties.